test/FsAutoComplete.Tests.Lsp/Tests.fs

```diff
@@ -502,15 +502,38 @@ let fsdnTest () =
       }
     ])
 
+let dotnetnewlistTest () =
+  let path = Path.Combine(__SOURCE_DIRECTORY__, "TestCases", "Empty")
+  serverTest path defaultConfigDto (fun (server, event) ->
+    do waitForWorkspaceFinishedParsing event
+
+    testList "DotnetNewList Tests" [
+      testAsync "DotnetNewList on list" {
+        let p : DotnetNewListRequest = {
+            Query = "Console Application"
+        }
+
+        let! res = server.FSharpDotnetNewList p
+        match res with
+        | Result.Error e -> failtestf "Request failed: %A" e
+        | Result.Ok n ->
+          Expect.stringContains n.Content "Console Application" (sprintf "the Console Application is a valid response, but was %A" n)
+          let r = JsonSerializer.readJson<CommandResponse.ResponseMsg<CommandResponse.DotnetNewListResponse>>(n.Content)
+          Expect.equal r.Kind "dotnetnewlist" (sprintf "dotnetnewlist response, but was %A, from json %A" r n)
+          Expect.contains r.Data.Functions "Console Application" (sprintf "the Console Application is a valid response, but was %A, from json %A" r n)
+      }
+    ])
+
 ///Global list of tests
 let tests =
    [
-    initTests
-    basicTests
-    codeLensTest
-    documentSymbolTest
-    autocompleteTest
-    renameTest
-    gotoTest
+    // initTests
```

Comment:
the comments should not be committed, or these should be reverted when this PR is ready

Reply:
Sure will keep that in mind to revert it.